src/messages/pt.json

```diff
@@ -7,20 +7,20 @@
   "landing_page_documentation.title": "Documentação",
   "documentation_start_here.title": "Comece aqui",
   "documentation_start_here.description": "Aprenda, passo a passo, tudo que precisa saber para começar a usar os principais módulos da VTEX.",
-  "documentation_tutorials.title": "Tutoriais",
-  "documentation_tutorials.description": "Acesse a documentação de todos os módulos da plataforma e tire suas dúvidas.",
-  "documentation_developers_portal.title": "Portal do desenvolvedor",
+  "documentation_tutorials.title": "Tutoriais hub",
+  "documentation_tutorials.description": "Navegue pelos nossos tutoriais para melhorar sua compreensão da plataforma.",
+  "documentation_developers_portal.title": "Portal do Desenvolvedor",
```

Comment:
why change this line? this had already been reviewed by loc

Reply:
This change is due to another modification occurring after the creation of this branch. I will restore the original information.